src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/DeleteExpenseCommand.java

```diff
@@ -14,15 +14,6 @@
  */
 public class DeleteExpenseCommand extends DeleteCommand {
 
-    public static final String COMMAND_WORD = "delete";
-
-    public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the expense identified by the index number used in the displayed expense list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
-
-    public static final String MESSAGE_DELETE_TRANSACTION_SUCCESS = "Deleted Expense: %1$s";
```

Comment:
Currently, these specific success messages are being shown. Should these be removed?

Reply:
Fixed